src/LogFileReader.cpp

```diff
@@ -13,13 +14,18 @@ LogFileReader::~LogFileReader()
 {
 }
 
+namespace
+{
+       QString TIME_OUTPUT = "hh:mm:ss.zzz";
```

Comment:
TIME_OUTPUT isn't really an output for time, but it converts a string to a QTime. The name should be changed to represent that.